docs/integrations/app-connections/aws.mdx

```diff
@@ -229,10 +231,10 @@ Infisical supports two methods for connecting to AWS.
                                     "secretsmanager:DescribeSecret",
                                     "secretsmanager:TagResource",
                                     "secretsmanager:UntagResource",
-                                    "kms:ListKeys",
-                                    "kms:ListAliases",
-                                    "kms:Encrypt",
-                                    "kms:Decrypt"
+                                    "kms:ListKeys", // if you need to specify the KMS key
+                                    "kms:ListAliases", // if you need to specify the KMS key
+                                    "kms:Encrypt", // if you need to specify the KMS key
```

Comment:
I was facing the issue below and the resolution was that I had to add the user manually to the KMS key user policy. might be helpful to mention that here?

<img width="968" alt="image" src="https://github.com/user-attachments/assets/adbcf9c8-8b98-410a-b442-84e534d103fe" />

Reply:
KMS key users UI: 
<img width="1236" alt="image" src="https://github.com/user-attachments/assets/379a1378-42b3-48fb-ba8b-527954fb7bf5" />

Reply:
not sure why this isn't an issue for aws parameter store though